modules/quota-increase/README.md

```diff
@@ -0,0 +1,60 @@
+# Request AWS Quota Increase
```

Comment:
NIT: I think this should be in our standard format with Learn, Deploy, Manage sections.

Reply:
Where can I find a example with this? The other modules don't follow this pattern :/

Reply:
Example: https://github.com/gruntwork-io/terraform-aws-eks/